frontends/web/src/components/sidebar/sidebar.tsx

```diff
@@ -203,7 +203,7 @@ class Sidebar extends Component<Props> {
           {
             accountsByKeystore.map(keystore => (<React.Fragment key={keystore.keystore.rootFingerprint}>
               <div className={style.sidebarHeaderContainer}>
-                <span className={style.sidebarHeader} hidden={!keystores?.length}>
+                <span className={style.sidebarHeader} hidden={!keystore.accounts.length}>
                   {t('sidebar.accounts')} - {keystore.keystore.name}
```

Comment:
Just a nit: I find the `ACCOUNT - <keystore name>` notation a bit confusing, as it seems to suggest that  <keystore name> is the name of an account. I'd prefer `<keystore name> - ACCOUNT` or even removing the ACCOUNT part completely (maybe replacing it with the root fingerprint? smth like `<keystore name> [fingerprint]`. 
Anyway, this can be discussed and possibly changed in a future PR as well.

cc @jadzeidan

Reply:
For clarity, this is the title of the group header above the accounts (it says ACCOUNTS, not ACCOUNT).

Anyway, I agree the "ACCOUNTS" part could be dropped.

About the fingeprint: we had discussed that fingeprints are too technical/noisy to show by default. that discussion was in relation to  disambiguate keystores from the same wallet but different passphrases, but decided that users can just rename the individual accounts.

Reply:
Ah then it is just in the Italian translation, I guess. Plural english words are often transformed into their singular version: in Italian we have 'ACCOUNT' instead of accounts.